docs/UserGuide.md

```diff
@@ -3,7 +3,7 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+Tutor Tracker is a **desktop app designed to help students search for tutors and manage tuition appointments**, optimised for use via a Command Line Interface (CLI) for a fast and streamlined experience.
```

Comment:
This update is no longer required as it was changed in another pull request, can revert to the current master revision.

Reply:
Noted! Will omit during the merge.